apps/anoma_node/lib/node/logging.ex

```diff
@@ -589,14 +565,18 @@ defmodule Anoma.Node.Logging do
     end
   end
 
-  @doc """
-  I am the name of the event table.
+  @spec init_table(String.t()) :: :ok
+  defp init_table(node_id) do
+    # initialize the tables
+    Tables.initialize_tables_for_node(node_id, @tables)
 
-  Given a Node ID, I create an appropriately named Event table for it.
-  """
+    # insert default record in the events table
+    table = Tables.node_table_name(node_id, @events_table)
+
```

Comment:
We should still clear the table here due to replay semantics. Either that or we should clear the table at the supervision level before launching the Node.

However since we already do a round write here, it seems better to internalize all of this minor operations in one spot. That way all "unnatural" operations to be done for the Event table will be done in one spot.